kmod/igb/igb_main.c

```diff
@@ -10408,6 +10407,11 @@ static long igb_mapbuf(struct file *file, void __user *arg, int ring)
 			       req.queue);
 			return -EINVAL;
 		}
+		
+		if(!adapter->num_tx_queues) {
+			printk("tx ring freed\n");
```

Comment:
To my way of thinking the text in this message (and the rx one) should be more verbose.